52/cod3Ghoul/pomodoro.py

```diff
@@ -0,0 +1,225 @@
+        len_of_pomos = int(input('\nEnter the length in minutes for each '
+                                 'pomodoro: '))
+        len_of_short_break = int(input('\nEnter the length in minutes for '
+                                       'each short break between '
+                                       'pomodoros: '))
+        len_of_long_break = int(input('\nEnter the length in minutes for '
+                                      'each long break after a full set of '
+                                      'pomodoros is completed: '))
+        while True:
+            print(f'\nYou have set your pomodoro timer up with the following '
+                  f'settings:\n'
+                  f'\n\tNumber of Pomodoros per set: {num_of_pomos}'
+                  f'\n\tLength of each Pomodoro: {len_of_pomos} '
+                  f'{"minute" if len_of_pomos == 1 else "minutes"}'
+                  f'\n\tLength of each Short Break: {len_of_short_break} '
+                  f'{"minute" if len_of_short_break == 1 else "minutes"}'
+                  f'\n\tLength of each Long Break: {len_of_long_break} '
+                  f'{"minute" if len_of_long_break == 1 else "minutes"}')
+            print('\n*As soon as you enter "confirm" at the prompt below, '
+                  'your first pomodoro will start using your '
+                  'chosen settings.')
+            confirm = input('\nTo confirm these settings enter "confirm" or '
+                            'to change them enter "change": ')
+            if confirm.lower() == 'confirm':
+                confirmed = True
+                break
+            elif confirm.lower() == 'change':
+                break
+            else:
+                system('clear')
+                print('\n\t** You did not enter "confirm" or "change" at the '
+                      'previous prompt. **')
+                sleep(3)
+                system('clear')
+    return [num_of_pomos, len_of_pomos, len_of_short_break, len_of_long_break]
+
+
+def start_pomodoro(settings):
+    global pomodoros
+    while True:
+        pomodoro_timer(settings[1])
+        if pomodoros < settings[0]:
+            short_break = input('Enter "b" to start your short break: ')
+            if short_break.lower() == 'b':
+                break_timer(settings[2])
+                print('\n> Enter "s" to start your next pomodoro.')
+        if pomodoros == settings[0]:
+            long_break = input('Enter "b" to start your long break: ')
+            if long_break.lower() == 'b':
+                break_timer(settings[3])
+                print(f'\n** You have completed a set of {settings[0]} '
+                      f'{"pomodoro" if settings[0] == 1 else "pomodoros"}.\n')
+        print('> Enter "r" to reset your pomodoro count '
+              'and start a new set of pomodoros using your '
+              'existing settings.\n'
+              '> Enter "m" to exit to the main menu.\n\n')
+        while True:
```

Comment:
wonder if we need a while in a while, makes it more complex, same for the nested if statements, consider refactoring (again just suggestions, happy to merge this weekend).